cli.js

```diff
@@ -101,10 +103,14 @@ process.stdin.on('keypress', (ch, key) => {
 		query.pop();
 	} else if (key.name === 'return' || (key.ctrl && key.name === 'u')) {
 		query.length = 0;
-	} else if (key.name === 'up' && skinNumber < 5) {
-		skinNumber++;
-	} else if (key.name === 'down' && skinNumber > 0) {
-		skinNumber--;
+	} else if (key.name === 'up') {
+		if (skinNumber < 5) {
+			skinNumber++;
+		}
+	} else if (key.name === 'down') {
+		if (skinNumber > 0) {
+			skinNumber--;
+		}
```

Comment:
I changed this because when pressing up/down when already at the max/min range, it would push undefined chars to the query, which don't appear or affect the fetching but would affect the backspace action.